UPGRADE-1.4.md

```diff
@@ -42,8 +40,53 @@ for more details:
     imports:
         - { resource: "services_test.yaml" }
     ```
-* If you use our Travis CI configuration, follow [theses changes](https://github.com/Sylius/Sylius-Standard/pull/323/files#diff-354f30a63fb0907d4ad57269548329e3) introduced in `.travis.yml` file
+* If you use our Travis CI configuration, follow [these](https://github.com/Sylius/Sylius-Standard/pull/322/files#diff-354f30a63fb0907d4ad57269548329e3)
+and [these](https://github.com/Sylius/Sylius-Standard/pull/323/files#diff-354f30a63fb0907d4ad57269548329e3) changes introduced in `.travis.yml` file
+
+---
+---
+
+# UPGRADE FROM `v1.2.X` TO `v1.4.0`
+
+The first step is changing your required php version to `^7.2` and then upgrading Sylius with Composer
```

Comment:
If the section is doubling the one higher, I'd just say to upgrade to v1.3 first (without directory change) and then to follow Doctrine Migrations upgrade guide above (so that we won't double the same instructions).

Reply:
I was thinking about such an approach, and I wasn't 100% sure about that... There are some differences in this and previous upgrade guide (doctrine migrations namespaces, adding bundle, importing file) but they can be just mentioned here, I suppose 😄 on it